.github/native-tests.json

```diff
@@ -9,7 +9,7 @@
         {
             "category": "Data1",
             "timeout": 65,
-            "test-modules": "jpa-h2, jpa-mariadb, jpa-mssql, jpa-derby, jpa-without-entity, hibernate-orm-tenancy/datasource",
+            "test-modules": "jpa-h2, jpa-mariadb, jpa-mssql, jpa-derby, jpa-without-entity, hibernate-orm-tenancy/datasource, hibernate-orm-tenancy/connection-resolver",
```

Comment:
Not sure I would have kept this commit separated from the one where you added the test, a bit too much to my taste :).

Reply:
To each their own :) I'll squash if I need to change this PR again.